esp/templates/program/modules/volunteersignup/signup.html

```diff
@@ -73,24 +152,73 @@ <h3>Volunteer Shifts Cancelled</h3>
 
 <div id="program_form">
 <form method="POST" id="volunteer_shift_form" action="{{ request.path }}">
+{% for hidden in form.hidden_fields %}
+    {{ hidden }}
+{% endfor %}
 <center>
-<table width="500">
-<tr>
-    <th colspan="2">Volunteer for {{ program.niceName }}</th>
-</tr>
-<tr>
-    <td colspan="2">{% if request.user.email %}Since you are logged in, your information is pre-filled but you can change it for the purposes of contacting you about volunteering.{% else %}If you have an account, please <a href="/myesp/login?next={{ request.path }}">click here</a> to log in and return to this page.{% endif %}</td>
-</tr>
+<p>{% if request.user.email %}Since you are logged in, your information is pre-filled but you can change it for the purposes of contacting you about volunteering.{% else %}If you have an account, please <a href="/myesp/login?next={{ request.path }}">click here</a> to log in and return to this page.{% endif %}</p>
 {% if form.errors %}
-<tr>
-    <td colspan="2"><span style="color: red;">Please correct the errors in the form.</span></td>
-</tr>
+<p><span style="color: red;">Please correct the errors in the form.</span></p>
 {% endif %}
-    {{ form }}
-<tr>
-    <td colspan="2" align="center"><input id="volunteer_form_submit" type="submit" value="Submit" class="btn btn-primary" /></td>
-</tr>
-</table>
+{{ form.name.label_tag }}
```

Comment:
Can you loop through the fields here instead of hardcoding?

Reply:
Hmm...maybe? Although if I loop through all fields it'll include the confirm field...

Reply:
Oh, this might work: https://stackoverflow.com/a/21698814/4660582